Build/PackageFiles/Dotnet_precompiled/ItemTemplates_v3.x.nuspec

```diff
@@ -45,5 +45,6 @@
     <file src="..\..\..\Functions.Templates\Templates\KafkaOutput-CSharp\**" target="content\KafkaOutput-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
     <file src="..\..\..\Functions.Templates\Templates\SqlInputBinding-CSharp\**" target="content\SqlInputBinding-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
     <file src="..\..\..\Functions.Templates\Templates\SqlOutputBinding-CSharp\**" target="content\SqlOutputBinding-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
+    <file src="..\..\..\Functions.Templates\Templates\EventGridBlobTrigger-CSharp-Isolated\**" target="content\EventGridBlobTrigger-CSharp" />

```

Comment:
Same here, only need to update ItemTemplates_v4

Reply:
Removed